src/main/java/me/shivzee/util/Message.java

```diff
@@ -207,6 +212,31 @@ public String getCreatedAt() {
         return createdAt;
     }
 
+    /**
+     * Get the Message Received Date/Time in ZonedDateTime format
+     * @return the date at which the message was sent/created/received
+     */
+    public ZonedDateTime getCreatedDateTime() {
+        return createdDateTime;
+    }
+
+    public void setCreatedDateTime(ZonedDateTime createdDateTime) {

```

Comment:
there is no need of a setter over here this class just binds the data and it should be immutable 
remove this function `setCreatedDateTime(ZonedDateTime)`